tutorial/detection/tag/tutorial-apriltag-detector.cpp

```diff
@@ -291,26 +291,23 @@ int main(int argc, const char **argv)
     }
     //! [Display camera pose for each tag]
 
-    vpDisplay::displayText(I, 20, 20, "Click to quit.", vpColor::red);
+    vpDisplay::displayText(I, 20, 20, "Click.", vpColor::red);
     vpDisplay::flush(I);
     vpDisplay::getClick(I);
     vpDisplay::close(I);
 
 
 #if (VISP_CXX_STANDARD >= VISP_CXX_STANDARD_11)
-    display2 = vpDisplayFactory::createDisplay(I_color, 50, 50);
+    display2 = vpDisplayFactory::createDisplay(I_color, 50, 50, "AprilTag detection (class display)");
 #else
-    display2 = vpDisplayFactory::allocateDisplay(I_color, 50, 50);
+    display2 = vpDisplayFactory::allocateDisplay(I_color, 50, 50, "AprilTag detection (class display)");
 #endif
     // To test the displays on a vpRGBa image
     vpDisplay::display(I_color);
 
-    // Display frames and tags but remove the display of the last element
+    // Display frames and tags
     std::vector<std::vector<vpImagePoint> > tagsCorners = detector.getTagsCorners();
-    tagsCorners.pop_back();
```

Comment:
Remove the `pop_back()`.
Will produce a segfault if there is no detection and it is not useful.